spec/models/prefix_spec.rb

```diff
@@ -11,7 +11,7 @@
   end
 
   describe "methods" do
-    it "prefixes all" do
+    it "prefixes all", :skip_prefix_pool do
```

Comment:
Shouldn't this be unaffected by any changes, as this is testing Prefixes not their relations?

Reply:
I will change it.  I was getting things based on the assumption there were no other prefixes available.  For example, this expected create_list to end up with 10 prefixes when it ended up with 30.  Then it expected the prefix to be 10.50821 and got 10.5081.  The :skip_prefix_pool was there to allow the tests to run as usual, and I believe they were just as valid.

However, I also think you are correct and we should just test with the prefix pool.  It should be unaffected.

Removing :skip_prefix_pool.